### PR TITLE
changes made by sortpom

### PR DESCRIPTION
**Pull request by mmusgrov**
There is no accompanying JBTM.
When I build the quickstarts sort pom is adding these changes.

NOTEST

### CODE REVIEW

**Comment by mmusgrov** [comment]
@jmfinelli Will you sanity check this one please?

File: XTS/pom.xml
Comment (jmfinelli) [inline XTS/pom.xml:19]: Just for consistency, I would remove the spacing, i.e.
```suggestion
    <!-- to enable ssl see https://issues.redhat.com/browse/JBTM-3851-->
```

File: XTS/pom.xml
Comment (jmfinelli) [inline XTS/pom.xml:20]:
```suggestion
    <!-- <module>ssl</module>-->
```

File: compensating-transactions/pom.xml
Comment (jmfinelli) [inline compensating-transactions/pom.xml:21]:
```suggestion
    <!-- <module>mongodb-simple</module> -->
```

**Review by jmfinelli — APPROVED** [review]
Apart from the suggested changes, this LGTM. Thanks

**Comment by mmusgrov** [comment]
I raised this PR because each time I run any quickstarts the sortpom plugin checks out the files and therefore I can't switch to alternate branches unless I discard the changes that sortpom made and it makes development a more long winded process. I didn't raise the PR to fix anything else apart from this annoying side effect of sortpom. Can we disable sortpom and request that engineers re-enable it whenever they change a pom?

**Comment by mmusgrov** [comment]
These errors are caused either by not running sortpom before merging the change that set `sortpom.skip` to false, or by temporarily disabling sortpom and then pushing a commit that causes sortpom to fail, there was a third alternative that I forget.

If you don't want to disable sortpom then an alternative would be to enable it on pull requests and fail the PR at that point.

**Comment by jmfinelli** [comment]
I'm not sure I fully understand what is/was the problem.

> Can we disable sortpom and request that engineers re-enable it whenever they change a pom?

I guess this is a change in our community procedures, which should require a wider discussion. Hence, I don't think I can take a decision on my own

**Comment by mmusgrov** [comment]
It's not a community issue, it is our decision. I have approval so I'll merge it.

File: XTS/pom.xml
Comment (mmusgrov) [reply XTS/pom.xml:19]: @jmfinelli I would like to point out that these changes were made by sortpom, not myself, and if I start adding extra changes then that would obfuscate the background to the change.

Comment (jmfinelli) [reply XTS/pom.xml:19]: I guess sortpom does not modify comments in the code. That would be up to us. If you prefer to leave things as are, it is fine for me

**Pull request merged**
